torchvision/transforms/_presets.py

```diff
@@ -37,21 +43,38 @@ def __init__(
         interpolation: InterpolationMode = InterpolationMode.BILINEAR,
     ) -> None:
         super().__init__()
-        self._crop_size = [crop_size]
-        self._size = [resize_size]
-        self._mean = list(mean)
-        self._std = list(std)
-        self._interpolation = interpolation
+        self.crop_size = [crop_size]
+        self.resize_size = [resize_size]
+        self.mean = list(mean)
+        self.std = list(std)
+        self.interpolation = interpolation
```

Comment:
Just removing the `_` because the whole class is private and marking the individual features seems an overkill. Plus it messes up the `__repr__` strings.